src/components/UpdateAppModal/BaseUpdateAppModal.tsx

```diff
@@ -27,7 +28,4 @@ function BaseUpdateAppModal({translate, onSubmit}) {
     );
 }
 
-BaseUpdateAppModal.propTypes = propTypes;
-BaseUpdateAppModal.defaultProps = defaultProps;
-
-export default memo(withLocalize(BaseUpdateAppModal));
+export default React.memo(BaseUpdateAppModal);
```

Comment:
What's about `displayName` for this component?

Reply:
It wasn't there before the migration, do you think we should add it?